DeepSpeech.ipynb

```diff
@@ -1270,6 +1273,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "For reporting we also need a standard way to do time measurements."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# This function will toggle a stopwatch\n",
+    "# The first call starts it, second call stops it, third call continues it etc.\n",
```

Comment:
This comment doesn't seem to describe what this actually does? Looks like a call with no parameter will return the current time and a call with a time will return how much time has elapsed since then.